docs/backends/dropbox.rst

```diff
@@ -26,5 +26,13 @@ To use DropBoxStorage set::
       Timeout in seconds for making requests to the API. If ``None``, the client will wait forever.
       The default is ``100`` seconds which is the current default in the official SDK.
 
+``DROPBOX_FILE_OVERWRITE`` (optional)
```

Comment:
No reason to include this. It's the same as setting `DROPBOX_WRITE_MODE = 'overwrite'`.